index.js

```diff
@@ -242,9 +304,6 @@ async function buildPlatforms() {
             }
 
             platformConfig.features[key] = { ...platformOverride.features[key] };
-            if (isFeatureMissingState(platformConfig.features[key])) {
```

Comment:
iirc this gets caught with schema now.
If the feature doesn't have a "state" in the features file and also not in the platform. Also it'd be deleted above.